src/Traits/Findable.php

```diff
@@ -2,9 +2,16 @@
 
 namespace HeadlessLaravel\Finders\Traits;
 
+use Illuminate\Support\Facades\Request;
+
 trait Findable
 {
     use Searchable;
     use Sortable;
     use Filterable;
+
+    public function getPerPage()
+    {
+        return Request::input('per_page', 15);
```

Comment:
This trait is only to add all the other traits

Reply:
Removed.